WordPress/Classes/Services/PeopleService.swift

```diff
@@ -119,33 +129,47 @@ struct PeopleService {
     /// - Returns: A new Person instance, with the new Role already assigned.
     ///
     func updateUser(_ user: User, role: String, failure: ((Error, User) -> Void)?) -> User {
-        guard let managedPerson = managedPersonFromPerson(user) else {
-            return user
+        var pristineRole: String?
+        var updated: User?
+        coreDataStack.performAndSave { context in
+            guard let managedPerson = managedPersonFromPerson(user, in: context) else {
+                return
+            }
+
+            // OP Reversal
```

Comment:
Do you know what "OP" stands for? Is it "operation"? "Optional"? Is this comment even useful?

Reply:
No idea, "optional" is my guess 🥲 

Not particularly useful I think? The code itself is quite clear.